.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Node.js CI
+
+on: [ push ]
```

Comment:
Does this mean the job will not run on pull requests?

Reply:
It does. I had both `push` and `pull_request` at first and it ends up running the test twice.

Reply:
Looked up a few repos and `vega-lite` use this for the tests. This should mean any push to any branch.

Reply:
`pull_request` is still different though because it runs on the merged code. If the PR branch didn't rebase before merge, the merged code could still break.

Reply:
umm, that makes sense now. i will add back the `pull_request`